nilearn/plotting/img_plotting.py

```diff
@@ -292,6 +295,10 @@ def plot_img(img, cut_coords=None, output_file=None, display_mode='ortho',
         Default=False.
     %(colorbar)s
         Default=False.
+    cbar_tick_format: str, optional
+        Controls how to format the tick labels of the colorbar.
+        Ex: use "%%i" to display as integers.
+        Default is '%%.2g' for scientific notation.
```

Comment:
Just curious: why is the % duplicated ?

Reply:
I believe due to the use of the `fill_doc` decorator, which applies `%` string formating to the docstring: https://github.com/nilearn/nilearn/blob/5cba192e3f1183330158f8d7469f36c01ba1f534/nilearn/_utils/docs.py#L790

Reply:
Absolutely! :)

Reply:
Thx